docs/source-pytorch/fabric/fabric.rst

```diff
@@ -589,3 +579,19 @@ For single-device strategies, it is a no-op. There are strategies that don't sup
 - xla
 
 For these, the context manager falls back to a no-op and emits a warning.
+
+
+call
+====
+
+Use this to run all registered callback hooks with a given name and inputs.
+It is useful when building a Trainer that allows the user to run arbitrary code at fixed points in the training loop.
+
+.. code-block:: python
+
+    fabric = Fabric(callbacks=[MyCallback()]))
+
+    # Call any hook by name
+    fabric.call("on_train_epoch_end", results={...})
```

Comment:
here it is not clear, why you pass in `results`. IMO you should include a dummy callback that at least shows the signature for `on_train_epoch_end` and mention this specifically. Or at least link to the callbacks section for more details

Reply:
Addressed!